packages/builder/src/components/portal/licensing/licensingBanners.js

```diff
@@ -146,23 +145,19 @@ const buildUsersAboveLimitBanner = EXPIRY_KEY => {
   const userLicensing = get(licensing)
   return {
     key: EXPIRY_KEY,
-    type: BANNER_TYPES.WARNING,
+    type: BANNER_TYPES.NEGATIVE,
     onChange: () => {
       defaultCacheFn(EXPIRY_KEY)
     },
     criteria: () => {
-      return userLicensing.warnUserLimit
+      return userLicensing.errUserLimit
     },
-    message: `${capitalise(
-      userLicensing.license.plan.type
-    )} plan changes - Users will be limited to ${
-      userLicensing.userLimit
-    } users in ${userLicensing.userLimitDays}`,
+    message: "Your Budibase account is de-activated. Upgrade your plan",
     ...{
-      extraButtonText: "Find out more",
+      extraButtonText: "View plans",
       extraButtonAction: () => {
         defaultCacheFn(ExpiringKeys.LICENSING_USERS_ABOVE_LIMIT_BANNER)
```

Comment:
Not sure about this. As far as I see, the `ExpiringKey` is a way to avoid to show the banner again and again, but it doesn't seems to be adapted to be shown for each user; once its shown to the first one, the rest admins will receive it?

Reply:
The expiration is stored in local storage, so that if you dismiss it you will see it again in 24 hours. If you log into another browser you should see it again.

Reply:
Ahh. Great. 👍